app/views/hyrax/file_sets/_actions.html.erb

```diff
@@ -1,4 +1,4 @@
-<% if can?(:edit, file_set.id) || can?(:destroy, file_set.id) || (Flipflop.enabled?(:annotation) && can?(:download, file_set.id)) %>
+<% if can?(:edit, file_set.id) || can?(:destroy, file_set.id) || Flipflop.enabled?(:annotation) && file_set.pdf? && can?(:download, file_set.id) %>
```

Comment:
Just checking out this condition

    Flipflop.enabled?(:annotation) && file_set.pdf? && can?(:download, file_set.id)
      
 Should   **can?(:download, file_set.id)**  not be something you can do even if the **file_set** is not a pdf provided it is not private.

Reply:
Ah yes.  You are correct.  There's a lot of conditions on this page and it's easy to get lost in them.

Reply:
Addressed below.